services/galley/galley.cabal

```diff
@@ -466,7 +466,6 @@ executable galley-schema
     ld-options: -static
   default-language: Haskell2010
 
--- FUTUREWORK(Leif): rename to galley-tests here and in the package.yaml file
```

Comment:
https://github.com/wireapp/wire-server/pull/1942  :)  (if you approve that PR, please merge right away.)